__tests__/groundplaces/updateStopCluster.spec.ts

```diff
@@ -52,7 +52,71 @@ describe('#updateStopCluster', () => {
     ]);
   });
 
+  it('should update the name of the StopCluster and avoid undefined values that can be pass', () => {
+    const groundPlacesService: GroundPlacesController = new GroundPlacesController();
+
+    groundPlacesService.init(mockSmallGroundPlacesFile as GroundPlacesFile);
+
+    groundPlacesService.updateStopCluster('c|FRstrasbou@u0ts2', {
+      name: 'Strasbourg, Est, France',
+      longitude: undefined,
```

Comment:
it should throw an error

Reply:
`longitude` ne sera pas pris en compte dans l'update de StopCluster, vu que cette valeur correponds à `undefined`, pourquoi throw une erreur ?

Reply:
Agreed with @Simobain to say that's weird to have some undefined value.

As a reminder, don't forget dans JSON isn't handling undefined values. I think that's not a good way to use undefined values.

Reply:
Ok en fait je faisait le travail inverse, c'est à dire accepter la valeur même si il est `undefined` et la nettoyer pour ne garder que les valeurs "corrects". Depuis vos retours, je gère maintenant dès l'entrée les cas où une des propriétés à update a comme valeur `undefined`, et je throw une valeur dans ce cas!